.github/workflows/helm.yml

```diff
@@ -0,0 +1,41 @@
+name: Helm CI
+on:
+  push:
+    paths:
+      - 'chart/**'
+      - '!chart/README.md'
+      - '!chart/.helmignore'
+  pull_request:
+    paths:
+      - 'chart/**'
+      - '!chart/README.md'
+      - '!chart/.helmignore'
+jobs:
+  lint:
+    if: github.ref != 'refs/heads/staging' && github.ref != 'refs/heads/production'
+    runs-on: ubuntu-latest
+    outputs:
+      changed: ${{ steps.lint.outputs.changed }}
```

Comment:
Are you adding this because you have separated `test` into its own job? Just a note that this isn't necessary if you keep these combined into a single `lint-test` job. Combined may also be a little faster as well, and less repeated step definitions.

Reply:
I initially had this in the same workflow as the ci.yml file. Fixing :)